README.md

```diff
@@ -38,6 +38,19 @@ workos.client_id = "client_1234"
 
 For our SDKs WorkOS follows a Semantic Versioning ([SemVer](https://semver.org/)) process where all releases will have a version X.Y.Z (like 1.0.0) pattern wherein Z would be a bug fix (e.g., 1.0.1), Y would be a minor release (1.1.0) and X would be a major release (2.0.0). We permit any breaking changes to only be released in major versions and strongly recommend reading changelogs before making any major version upgrades.
 
+## Beta Releases
+
+WorkOS has features in Beta that can be accessed via Beta releases. We would love for you to try these
+and share feedback with us before these features reach the stable phase. To install a Beta version, please follow the
```

Comment:
is there a more generic terminology for "stable phase"? like "general audience (GA)" or something like that?